todo-app.css

```diff
@@ -0,0 +1,90 @@
+:root {
+  --purple-color: rgba(112, 35, 112, 1);
+  --gray-color: rgb(120, 120, 120);
+  --light-purple-color: rgba(238, 212, 238, 1);
+}
+
+.heading-1 {
```

Comment:
`1` doesn't clearly say what heading is being styled here. Probably something more relatable like `main-heading` would be clear